src/app/lib/config/services/afrique.ts

```diff
@@ -288,6 +289,10 @@ export const service: DefaultServiceConfig = {
     },
     recommendations: {
       hasStoryRecommendations: false,
```

Comment:
Should we set these to `true` now or will that be a follow up PR?

Reply:
Hmm yeah not sure, wording on the PR says they want toggles to be the source of truth, so I guess yes? Although does that mean we can get rid of this in-code value altogether?

Reply:
Possibly worth consolidating if possible. The `hasStoryRecommendations` is used in 2 key places:

1) In a check before recommendations are fetched 
https://github.com/bbc/simorgh/blob/bb76dab841723ab95fd56c63e4feca103ef886b8/src/app/routes/cpsAsset/utils/hasRecommendations.js#L3-L4 

2) In the recommendations component to determine display
https://github.com/bbc/simorgh/blob/9997384bcad94b0175a7b8123b373788492f0183/src/app/legacy/containers/CpsRecommendations/index.jsx#L61-L63

The second one could probably be removed safety and rely entirely on the `enabled` toggle flag. The first one is a bit trickier as we'd need to pass in the toggles into that function, if they are available at that point in the page request.

Reply:
Cool, I'll have a gander and see if I can get a viable way for it to work

Reply:
Cheers 🙌🏼 No probs if it becomes a bit too complex for this though. We can follow the pattern we have currently if its too much of a refactor. It could also be worth just changing the `if (!hasStoryRecommendations || !enabled || !items.length) return null; ` logic to require the both `hasStoryRecommendations` and the `enabled` toggle to be true.

Reply:
Decided to just leave it in the end, did a bit of digging though, it is possible as we [fetch the toggles first before the page data](https://github.com/bbc/simorgh/blob/7ce5c79baff7ff559602dac7feb295c9fb049797/src/server/index.jsx#L203-L213).

Reply:
Thanks for taking a look 👍🏼 I've asked if there is any FABL module/service that we can use for fetching recommendations in the BFF. One to think about how we may integrate toggles into FABL as well, if its possible.